cli/interactive_tests/common.tcl

```diff
@@ -0,0 +1,38 @@
+# We are sending vt100 terminal sequences below, so inform readline
+# accordingly.
+set env(TERM) vt100
+
+# Keep the history in a test location, so as to not override the
+# developer's own history file.
+set histfile "/crl-history"
```

Comment:
If we can write to `/`, then we must be in docker. If we're in docker, then our home directory is not the user's home directory, so it should be fine to leave this as the default, right?